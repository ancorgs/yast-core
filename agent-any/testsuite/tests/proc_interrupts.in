           CPU0       
  0:     577701          XT-PIC  timer
  1:      35877          XT-PIC  keyboard
  2:          0          XT-PIC  cascade
 10:      80333          XT-PIC  aic7xxx, eth0
 12:      33819          XT-PIC  PS/2 Mouse
 13:          1          XT-PIC  fpu
NMI:          0
