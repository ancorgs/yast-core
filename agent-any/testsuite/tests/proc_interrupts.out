($["0":["577701", "XT-PIC", "timer"], "1":["35877", "XT-PIC", "keyboard"], "10":["80333", "XT-PIC", "aic7xxx", "eth0"], "12":["33819", "XT-PIC", "PS/2", "Mouse"], "13":["1", "XT-PIC", "fpu"], "2":["0", "XT-PIC", "cascade"], "nmi":0])
